surefire-its/src/test/java/org/apache/maven/surefire/its/JUnit47RedirectOutputIT.java

```diff
@@ -46,11 +45,11 @@ public void testClassesParallel() {
     }
 
     private void checkReports(OutputValidator validator) {
-        String report = StringUtils.trimToNull(validator
+        String report = trimToNull(validator
                 .getSurefireReportsFile("junit47ConsoleOutput.Test1-output.txt")
                 .readFileToString());
         assertNotNull(report);
-        String report2 = StringUtils.trimToNull(validator
+        String report2 = trimToNull(validator
```

Comment:
This is test code. The output and input should be fully reproducible as either null or not null, so we likely don't need any of this empty to null conversion here. Depending on what the output is either simply remove trimToNull, or remove trimToNull and expect an empty string instead of null. Much simpler.

Reply:
Agree. Code adjusted and pushed.